build/AzureInfo.props

```diff
@@ -1,6 +1,10 @@
 <Project>
   <PropertyGroup>
     <Product>Sdk</Product>
+    <AssetDownloadRootUrl>$(PB_AssetRootUrl)</AssetDownloadRootUrl>
+    <AssetDownloadRootUrl Condition="'$(AssetDownloadRootUrl)' == ''">https://dotnetcli.azureedge.net</AssetDownloadRootUrl>
+    <PrivateBlobAccessToken>$(PB_AssetRootAccessTokenSuffix)</PrivateBlobAccessToken>
+    <PrivateBlobAccessTokenParam Condition="'$(PrivateBlobAccessToken)' != ''">?$(PrivateBlobAccessToken)</PrivateBlobAccessTokenParam>
```

Comment:
I think you should keep the PrivateBlobAccessTokenParam settable through CoreSetupBlobAccessTokenParam. As it is, you are breaking the MSRC build definitions.